.github/workflows/radxa-cm3-io-rk3566.yml

```diff
@@ -0,0 +1,50 @@
+name: Radxa CM3 on Raspberry Pi Compute Module 4 IO Board
+
+on:
+  # https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore
+  # https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+  pull_request:
+    branches:
+      - main
+      - master
+      # ESR branches glob pattern
+      #- 20[0-9][0-9].[0-1]?[1470].x
+  # pull_request_target:
+  #   branches:
+  #     - main
+  #     - master
+  push:
+    tags:
+      # Semver tags glob pattern (includes ESR in format v20YY.MM.PATCH)
+      - v[0-9]+.[0-9]+.[0-9]+\+?r?e?v?*
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      force-finalize:
+        description: Force finalize of the build (implicitly enables hostapp and S3 deployments)
+        required: false
+        type: boolean
+        default: false
+      deploy-environment:
+        description: Environment to use for build and deploy
+        required: false
+        type: string
+        default: balena-staging.com
```

Comment:
We should expose `meta-balena-ref` like we did [here](https://github.com/balena-os/balena-generic/pull/538/files)